src/FSharpVSPowerTools.Core/LanguageService.fs

```diff
@@ -535,7 +535,7 @@ type LanguageService (?fileSystem: IFileSystem) =
                         { su with IsUsed = notUsedSymbols |> Array.forall (fun s -> not (s.IsEffectivelySameAs su.SymbolUse.Symbol)) })
         }
 
-    member x.GetAllEntitiesInProjectAndReferencedAssemblies (projectOptions: FSharpProjectOptions, fileName, source) =
+    member x.GetAllEntitiesInProjectAndReferencedAssemblies withCache (projectOptions: FSharpProjectOptions, fileName, source) =
```

Comment:
Could we make `withCache` the last and optional argument of `GetAllEntitiesInProjectAndReferencedAssemblies`? We pass `withCache` three times and in twice of those, `withCache` does nothing.